### DISCUSSION TITLE
Issue with Adding College Email to My GitHub Account

### DISCUSSION THREAD

**Discussion by ahmedeldamaty20** [community/community#134531] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I'm attempting to associate my college email address with my GitHub account, but I'm receiving an error message: 'Error adding: email is already in use.' I have not previously linked this email address to any GitHub account. What could be causing this issue?

**Comment by lukaspestalozzi** [comment]
see https://github.com/orgs/community/discussions/73524
